lib/data_migrate/database_tasks.rb

```diff
@@ -68,28 +68,23 @@ def run_migration(migration, direction)
         end
       end
 
-      def schema_dump_path(db_config, format = ActiveRecord.schema_format)
-        return ENV["DATA_SCHEMA"] if ENV["DATA_SCHEMA"]
+      unless Gem::Dependency.new("railties", "< 7.0").match?("railties", Gem.loaded_specs["railties"].version)
```

Comment:
I've re-extracted this check to make it more intent that we only add these two methods for newer Rails versions that have more complex support for multiple databases. All the previous gem versions compatible with rails 6.0 specifically worked without these methods.